pyproject.toml

```diff
@@ -22,16 +22,16 @@ pytest-mock = "3.12.0"
 aiohttp = "3.9.5"
 async-lru = "2.0.4"
 boto3 = "^1.28.56"
-fastapi = "0.109"
+fastapi = "0.115.0"
 fastapi-versionizer = "3.0.4"
 faster-whisper = "1.0.3"
-prometheus-client = "0.19.0"
-prometheus-fastapi-instrumentator = "6.1.0"
+prometheus-client = "0.20.0"
+prometheus-fastapi-instrumentator = "7.0.0"
 pyjwt = {extras = ["crypto"], version = "^2.8.0"}
 python = "~3.11"
 redis = "5.0.1"
-torch = ">=2.0.0,<2.1.0"
-torchaudio = ">=2.0.1,<2.1.0"
+torch = "2.4.0"
+torchaudio = "2.4.0"
```

Comment:
@rpurdel i tested this with the github demo page and it seems to be working, anything else I could do to ensure I'm not breaking stuff?

Reply:
tested on A10, seems to be working fine